akka-docs/rst/java/code/docs/http/javadsl/server/directives/FutureDirectivesExamplesTest.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
```

Comment:
@Hawstein That should be nice too,just not that consistent,anyway ,we have git to track.